### PR TITLE
Update Assert-False.Tests.ps1

### PR DESCRIPTION
**Pull request by irwins**
Edited some Test cases description to give it more meaning...

Rg./Irwin

### CODE REVIEW

File: tst/Boolean/Assert-False.Tests.ps1 [hunk 1]
```diff
@@ -1,13 +1,13 @@
 Describe "Assert-False" {
-        It "Passes when given `$false" {
```
Comment (nohwnd) [inline tst/Boolean/Assert-False.Tests.ps1]: the original text is imho just fine. The test could not be simpler, and there is no need to state the name of the assertion in the test description when we have it in describe.

File: tst/Boolean/Assert-False.Tests.ps1 [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+        It "Passes when cmdlet Assert-False is given `$false" {
             $false | Assert-False
         }
 
-        It "Passes when given falsy" -TestCases @(
```
Comment (nohwnd) [inline tst/Boolean/Assert-False.Tests.ps1]: I like expanding the value in the description, but doing it from the actual value is just fine (use <actual> btw not <Actual> if you will fix it). Pester won't handle it correctly for the moment but I am okay with that. There are few other tests that would need to be changed as well, but I will rather focus on getting the functionality that shows the data correctly in pester, than manually writing the values out in extra key in hashtable. Again stating the name in the test name is not necessary.

File: tst/Boolean/Assert-False.Tests.ps1 [hunk 3]
```diff
@@ -23,7 +23,7 @@ Describe "Assert-False" {
     }
 
     Context "Validate messages" {
-        It "Given value '<expected>' that is not `$false it returns expected message '<message>'" -TestCases @(
```
Comment (nohwnd) [inline tst/Boolean/Assert-False.Tests.ps1]: you are correct here, the template should display Actual and not expected. But please use <actual> with lowecase letter.

File: tst/Boolean/Assert-False.Tests.ps1 [hunk 4]
```diff
@@ -33,7 +33,7 @@ Describe "Assert-False" {
         }
     }
 
-    It "Returns the value on output" {
```
Comment (nohwnd) [inline tst/Boolean/Assert-False.Tests.ps1]: no need to fix my quotes. it's stylistic, and I mostly use doubles so I can add templates easily.

**Comment by nohwnd** [comment]
@irwins thanks for finding my mistakes :) , added some comments to the PR. Please change the code a bit. Or I can cherry pick it and merge it myself if you don't want to bother with it.

**Comment by irwins** [comment]
@nohwnd No problem! Good stuff! Just trying to help out... The more eyes you have... I'm on vacation at the moment... So... ;-) Tested this on my MacBook Pro! Keep up the good work!!!

Rg./Irwin

**Pull request merged**